src/importexport/mei/internal/meiconverter.cpp

```diff
@@ -890,6 +891,11 @@ libmei::Clef Convert::clefToMEI(engraving::ClefType clef)
     case (engraving::ClefType::C4):
     case (engraving::ClefType::C4_8VB):
     case (engraving::ClefType::C5):
+    case (engraving::ClefType::C1_PETR):
+    case (engraving::ClefType::C2_PETR):
+    case (engraving::ClefType::C3_PETR):
+    case (engraving::ClefType::C4_PETR):
+    case (engraving::ClefType::C5_PETR):
```

Comment:
That's not a proper solution here. You should adjust the default case further down: just check if the `glyphName` starts with "mensural" and subtract it from the string.

Reply:
And convert the then leading uppercase C, F, G into lowercase?

Reply:
That'd also match the otherwise yet unhandled `mensuralGclef`, `mensuralFclef`, `mensuralCclef`, but still not `chantFclef` and `chantCclef` from [that range](https://www.w3.org/2019/03/smufl13/tables/medieval-and-renaissance-clefs.html)